gpu-jobs.md

```diff
@@ -166,115 +89,170 @@ server, including:
-```
-{: .sub}
- 
-It is important to still request at least one CPU per job to do the
-processing that is not well-suited to the GPU.
-
-Note that HTCondor will make sure your job has access to the GPU -- you
-shouldn't need to set any environmental variables or other options
-related to the GPU, except what is needed inside your code.
-
-## B. Use the GPU Lab Servers (recommended)
 
-To accept the CHTC GPU Lab policies and opt-in to use these GPUs, add the
-following line to your submit file:
+<table class="gtable">
+  <tr>
+    <th>Number of Servers</th>
+    <th>Names</th>
+    <th>GPUs / Server</th>
+    <th>GPU Type</th>
+    <th>CUDACapability</th>
+    <th>HasCHTCStaging</th>
+  </tr>
+<!--  <tr>
+    <td>gpu-3.chtc.wisc.edu</td> 
+    <td>1 </td>
+    <td>Tesla K40c</td>
+  </tr>   -->  
+  <tr>
+    <td>2</td>
+    <td>gpu2000, gpu2001</td>
+    <td>2</td>
+    <td>Tesla P100-PCIE-16GB</td>
+    <td>6.0</td>
+    <td>yes</td>
+  </tr>
+  <tr>
+    <td>4</td>
+    <td>gpulab2000 - gpulab2003</td>
+    <td>8</td>
+    <td>GeForce RTX 2080 Ti</td>
+    <td>7.5</td>
+    <td>yes</td>
+  </tr>
+</table>
 
-```
-+WantGPULab = true
-```
-{: .sub}
+## 2. Special Policies in the GPU Lab
 
-**REMINDER**: As described [above](#job-types-runtimes-and-per-user-limitations), using 
-the GPU Lab servers means that default CHTC job limits 
-are changed, including the length of the job. 
-To specify the job type, indicate which category of job length you would like 
-to use for your job, using the syntax described below: 
+Because the CHTC GPU Lab was
+funded by a special investment from the [UW2020 program](https://research.wisc.edu/funding/uw2020/round-5-projects/enabling-graphics-processing-unit-based-data-science/), we
+have implemented special policies to maximize how many researchers can 
+benefit from this investment. **Specifically, jobs running on GPU Lab servers have time limits and job number limits 
+(differing from CHTC defaults across the rest of the HTC System).**
 
 {:.gtable}
-  | Job type | Maximum runtime | Per-user limitation | Submit file flag | 
+  | Job type | Maximum runtime | Per-user limitation | 
   | --- |
-  | Short | 12 hrs | 2/3 of CTHC GPU Lab GPUs | `+GPUJobLength = "short"` | 
-  | Medium | 24 hrs | 1/3 of CTHC GPU Lab GPUs |  `+GPUJobLength = "medium"` | 
-  | Long  | 7 days | 1 job with 1-2 GPUs |  `+GPUJobLength = "long"` | 
+  | Short | 12 hrs | 2/3 of CTHC GPU Lab GPUs | 
+  | Medium | 24 hrs | 1/3 of CTHC GPU Lab GPUs |  
+  | Long  | 7 days | 1 job with 1-2 GPUs | 
+
+Each server in the GPU Lab has a single GPU slot reserved for interactive use. Interactive 
+jobs that use GPU Lab servers are restricted to using a single GPU and a 4 hour runtime. 
+
+These job types, runtimes, and per-user limitations are subject to change with
+short notice as the CHTC GPU Lab studies usage patterns.
 
-If you do not specify a job type, the `medium` job type will be used as the default.
+## 3. Submit Jobs Using the CHTC GPU Lab
```

Comment:
It makes sense to me to lead with this because we assume that most users will want to opt-in to the GPU Lab.  Should we add a forward reference to the `Access Additional GPU Resources` section here to alert people to the other resources that are available if they do not want to opt-in to GPU Lab?

Reply:
I added something a bit further up.